zh/deploy-tiflash.md

```diff
@@ -79,6 +79,44 @@ TiFlash 支持挂载多个 PV，如果要为 TiFlash 配置多个 PV，可以在
 
 如果服务器没有外网，请参考[部署 TiDB 集群](deploy-on-general-kubernetes.md#部署-tidb-集群)在有外网的机器上将用到的 Docker 镜像下载下来并上传到服务器上。
 
+## 移除 TiFlash
+
+1. 删除同步到 TiFlash 的数据表。
+
+    由于移除 TiFlash 后，TiFlash 集群剩余 Pod 数将为0，因此需要将 TiFlash 集群中所有同步数据表的副本数都置为0，才能完全移除 TiFlash。参考[访问 TiDB 集群](access-tidb.md)的步骤连接到 TiDB 服务，并使用以下命令删除 同步到 TiFlash 集群中的数据表。
+
+      {{< copyable "sql" >}}
+
+      ```sql
+      alter table <db-name>.<table-name> set tiflash replica 0;
```

Comment:
```suggestion
      alter table <db_name>.<table_name> set tiflash replica 0;
```

Reply:
Please also help update the command in the doc https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/scale-a-tidb-cluster#%E7%BC%A9%E5%AE%B9-tiflash.

Reply:
fixed